### ISSUE TITLE
uPortal Contrib?

### ISSUE THREAD

**Issue by ChristianMurphy** [mrbenshef/uPortal-auto-screenshot#3]
Hey @mrbenshef! :wave: 
Welcome the the uPortal community! :tada: 

This is a cool project.
There has been some interest in automated screenshot testing and accessibility testing in the community for a while now.
If you would be interested, this is a project that could be a good fit for [uPortal-contrib](https://github.com/uPortal-contrib).
uPortal contrib is a collection of community projects around uPortal, the GitHub organization gives a platform for multiple Universities to collaborate on uPortal related tools.
If you're interested there are more details at https://uportal-contrib.github.io/intake-process.html
The short version is, to become a part of contrib the project would need to reach out to [uPortal-dev](https://groups.google.com/a/apereo.org/forum/#!forum/uportal-dev) with:

1. A proposed name for the project, together with a brief explanation of the project 
2. A project readme
3. A proposed Project Lead with contact information
4. A list of the initial committers for the Project

If you have questions feel free to reach out to myself or any of the people listed in the CC.

Whatever your decision may be, thanks for making an awesome project!

/cc @asgrant @bjagg @cousquer @jimhelwig

**Comment by zoeyfyi** [comment]
I wrote this to get some initial screenshots for the marketplace and a web component I am working on. I hadn't thought about testing as a use case, but it makes a lot of sense. uPortal contrib sounds great, ill get in touch!

**Comment by zoeyfyi** [comment]
@ChristianMurphy whats the next step with this?

**Comment by ChristianMurphy** [comment]
Next step would be transferring the repo to uPortal contrib.
You'll need to be a member of uPortal-contrib to transfer the repo.
The steering committee is the only team that can grant access (as far as I'm aware).
/cc @apetro @asgrant @bjagg @cousquer @jimhelwig

**Comment by ChristianMurphy** [comment]
Added to this uPortal steering committee agenda, hopefully will get @mrbenshef access to uPortal contrib soon.

**Comment by ChristianMurphy** [comment]
This has been mirrored to https://github.com/uPortal-contrib/uPortal-auto-screenshot
You should have a pending invite to the repository.

Thanks for contributing this to the uPortal community!
